xml/sip_user_agents.xml

```diff
@@ -673,6 +673,19 @@
     <param pos="0" name="hw.device" value="VoIP Gateway"/>
   </fingerprint>
 
+  <fingerprint pattern="^(?:Eltex )?(?:smg_pa_sip[ -]){1,2}([\d.]+)$">
+    <description>Eltex SMG model VoIP gateway - no model number</description>
+    <example os.version="3.9.1.50">Eltex smg_pa_sip smg_pa_sip-3.9.1.50</example>
+    <example os.version="3.10.1.22">smg_pa_sip smg_pa_sip-3.10.1.22</example>
+    <example os.version="3.18.0.67">smg_pa_sip 3.18.0.67</example>
+    <param pos="0" name="os.vendor" value="Eltex"/>
+    <param pos="0" name="os.product" value="SMG"/>
```

Comment:
This should probably be `SMG firmware` to maintain consistency with other embedded / networking devices.